MCNPInput.cpp

```diff
@@ -1273,16 +1275,34 @@ void InputDeck::parseSurfaces( LineExtractor& lines ){
     if( do_line_continuation( lines, token_buffer ) ){
       continue;
     }
-   
-    SurfaceCard* s = new SurfaceCard(*this, token_buffer);
-
-    if( OPT_VERBOSE) s->print(std::cout);
+    int numFacets = 0;
+    
+    if(token_buffer.at(1).find_first_of("1234567890-") != 0){
```

Comment:
this seems to be addressing something other than macrobody facets - is this a different bug?